Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -3482,7 +3488,7 @@ std::optional<SimpleRange> AXObjectCache::rangeMatchesTextNearRange(const Simple
 
 static bool isReplacedNodeOrBR(Node& node)
 {
-    return AccessibilityObject::replacedNodeNeedsCharacter(node) || node.hasTagName(brTag);
+    return AccessibilityObject::replacedNodeNeedsCharacter(node) || WebCore::elementName(&node) == ElementName::HTML_br;
```

Comment:
Maybe we could have an `elementName` function that takes a `Node&` too, since this is now going to incur an extra null-check in the cast to element. We use this pattern elsewhere, e.g. for `hasAnyRole`:

https://github.com/WebKit/WebKit/blob/798647db3b710d3f5282bbc367a44bcc44200173/Source/WebCore/accessibility/AXObjectCache.cpp#L663#L666